pixel_clusterizer/clusterizer.py

```diff
@@ -9,11 +9,25 @@
                           ('charge', '<u2')])
 
 
+def np_uint_type_chooser(number):
```

Comment:
Sparse arrays have to be looped (O(n), less speed, less memory; lets ignore fancy ideas we will not implement) and dense arrays can be accessed in O(1) (more speed, more memory)